### PR TITLE
Remove app style colors

### PR DESCRIPTION
**Pull request by eviltofu**
# Commits
- [ ] [01 Removed Colors.alabaster](https://github.com/AlphaWallet/alpha-wallet-ios/pull/5956#issuecomment-1369341654)
- [ ] [02 Removed Colors.appActionButtonGreen](https://github.com/AlphaWallet/alpha-wallet-ios/pull/5956#issuecomment-1369342069)
- [ ] [03 Removed Colors.appActionButtonShadow](https://github.com/AlphaWallet/alpha-wallet-ios/pull/5956#issuecomment-1369342458)
- [ ] [04 Removed Colors.appBackground](https://github.com/AlphaWallet/alpha-wallet-ios/pull/5956#issuecomment-1369342458)
- [ ] [05 Removed Colors.appGrayLabel](https://github.com/AlphaWallet/alpha-wallet-ios/pull/5956#issuecomment-1369342837)
- [ ] [06 Removed Colors.appGreenContrastBackground](https://github.com/AlphaWallet/alpha-wallet-ios/pull/5956#issuecomment-1369342954)
- [ ] [07 Removed Colors.appHighlightGreen](https://github.com/AlphaWallet/alpha-wallet-ios/pull/5956#issuecomment-1369343076)
- [ ] [08 Removed Colors.appRed](https://github.com/AlphaWallet/alpha-wallet-ios/pull/5956#issuecomment-1369343202)
- [ ] Removed Colors.appSubtitle
- [ ] Removed Colors.settingsSubtitle
- [ ] Removed Colors.clear
- [ ] Removed Colors.loadingBackground
- [ ] Removed Colors.loadingIndicatorBorder
- [ ] Removed Colors.disabledActionButton
- [ ] Removed Colors.qrCodeRectBorders
- [ ] Removed Colors.navigationTitleColor
- [ ] Removed Colors.green
- [ ] Removed Colors.concrete
- [ ] Removed Colors.dove
- [ ] Removed Colors.lightGray
- [ ] Removed Colors.darkGray
- [ ] Removed Colors.gray
- [ ] Removed Colors.lightBlack
- [ ] Removed Colors.black
- [ ] Remove Colors.appText
- [ ] Remove Colors.appTint
- [ ] Remove Colors.appWhite
- [ ] Removed Colors

# Missing screens
[Missing](https://github.com/AlphaWallet/alpha-wallet-ios/pull/5956#issuecomment-1357039598)
I am unable to locate in the app where the screens linked above are.

### CODE REVIEW

**Review by hboon — COMMENTED** [review]
I think the files changed GitHub to add code owners as reviewers. Let us know if you want to check anything

**Comment by eviltofu** [comment]
> I think the files changed GitHub to add code owners as reviewers. Let us know if you want to check anything

I didn't request any reviewers for this. So Github does this automatically now?

**Comment by hboon** [comment]
We marked certain files as requiring review. Refer to the .github directory at the root. That’s why I asked.

**Comment by eviltofu** [comment]
- [ ] ButtonStyle.borderColor in BrowserErrorView

**Comment by hboon** [comment]
> Not Complete. How do I get these views to show up?
> 
> * [ ]  `ButtonStyle.borderColor` / `BrowserErrorView` / `BrowserViewController` / `DappBrowserCoordinator`.

ButtonStyle.borderColor
---
Look for `HelpUsCoordinator.wellDone()` and make sure it's called by commenting out the conditions around it

BrowserErrorView
---
Enter an invalid URL into the Browser tab URL bar, hit enter and you'll see it. e.g.: 127.0.0.1

BrowserViewController
---
This is the Browser tab

DappBrowserCoordinator
---
This creates a manage the Browser tab

**Comment by eviltofu** [comment]
`Delete this` will be deleted when I squish everything.

File: AlphaWallet/Common/Types/Configuration.swift [hunk 1]
```diff
@@ -274,14 +305,20 @@ struct Configuration {
...
+            static let dangerBackground = R.color.danger()!
+            static let appreciation = UIColor(red: 117, green: 185, blue: 67)
+            static let depreciation = UIColor(hex: "ff3b30")
+            static let pass = appreciation
+            static let fail = depreciation
+
             static let border = UIColor(red: 194, green: 194, blue: 194)
-            static let text = Colors.appText
-            static let label = Colors.appGrayLabel
+            // static let text = Colors.appText
```
[the remark this answers is not in the document]
Comment (eviltofu) [reply AlphaWallet/Common/Types/Configuration.swift]: Removed

**Comment by hboon** [comment]
FYI, if you modified https://github.com/AlphaWallet/alpha-wallet-ios/pull/5956#issuecomment-1357039598 and want me to check again, you'll have to buzz me. I don't think it can be discovered.

**Comment by hboon** [comment]
> actionButtonBackgroundColor in GenerateSellMagicLinkViewModel in <GenerateSellMagicLinkViewController> / NFTCollectionCoordinator

Similarly to the previous one, it's easier to modify the code for this particular case. In `NFTAssetViewController.actionButtonTapped`, change the body so it's just a 1-liner:

```
delegate?.didPressSell(tokenHolder: viewModel.tokenHolder, for: .send(type: .transaction(viewModel.sellTransactionType)), in: self)
```

Launch the app and go:

1. Settings > Advanced > Features > enable is Nft Transfer Enabled
2. Wallet tab
3. Collectibles
4. Tap a collectible
5. Tap an instance
6. Tap Send
7. Enter amount of ETH (say 1)
8. Tap Next
9. Tap Next again

Some of the UI might look broken. But I think it's because we modified the code to follow a different path

**Comment by eviltofu** [comment]
> FYI, if you modified [#5956 (comment)](https://github.com/AlphaWallet/alpha-wallet-ios/pull/5956#issuecomment-1357039598) and want me to check again, you'll have to buzz me. I don't think it can be discovered.

Maybe I'll buzz you when I've completed the list for the day.

**Comment by hboon** [comment]
> > FYI, if you modified [#5956 (comment)](https://github.com/AlphaWallet/alpha-wallet-ios/pull/5956#issuecomment-1357039598) and want me to check again, you'll have to buzz me. I don't think it can be discovered.
> 
> Maybe I'll buzz you when I've completed the list for the day.

Sure.

**Comment by eviltofu** [comment]
> > actionButtonBackgroundColor in StatusViewControllerViewModel  in  --- Suspect this one is no longer in use I did not find a coordinator or view controller which spawns this (via Find Symbol in Xcode)
> 
> Yes, looks like we should remove it. I'm inclined to fix it before removing it, but looks like it's using `UIButton` directly instead of a button bar, so let's not waste time on fixing it.

I commented out StatusViewControllerViewModel and StatusViewController and the app was able to compile without any issues. Want me to remove the files here in this PR?

**Comment by eviltofu** [comment]
WhereIsWalletAddressFoundOverlayView appears not to be in use. I commented out https://github.com/AlphaWallet/alpha-wallet-ios/blob/e1c75495b1cc13df3180f16e4e89818de0f06353/AlphaWallet/AppCoordinator.swift#L467 , https://github.com/AlphaWallet/alpha-wallet-ios/blob/6024eebe5d16100a354249fb5a204a20fb776210/AlphaWallet/Wallet/Coordinators/InitialNetworkSelectionCoordinator.swift#L11 , https://github.com/AlphaWallet/alpha-wallet-ios/blob/6024eebe5d16100a354249fb5a204a20fb776210/AlphaWallet/Wallet/Coordinators/InitialNetworkSelectionCoordinator.swift#L15 and was able to compile the app.

**Comment by hboon** [comment]
> WhereIsWalletAddressFoundOverlayView appears not to be in use. I commented out

Ah. Because we removed the network selection screen for new installs recently. Can you create a separate issue/PR to remove `InitialNetworkSelectionCoordinator` and friends but keep `WhereIsWalletAddressFoundOverlayView.swift` so we can add in a call to `WhereIsWalletAddressFoundOverlayView.show()` somewhere later. If it's still unused after a while, we'll remove it.

**Comment by eviltofu** [comment]
# Colors.alabaster

## ButtonStyle.borderColor

1. From main wallet screen, tap browser button.
2. Enter 127.0.0.1 in address bar of browser.

Class | Before | Light After | Dark After
-|-|-|-
ButtonStyle.borderColor / BrowserErrorView / BrowserViewController / DappBrowserCoordinator |Simulator Screen Shot - iPhone 14 - 2022-12-20 at 07 55 13|Simulator Screen Shot - iPhone 14 Pro - 2022-12-20 at 09 40 54|Simulator Screen Shot - iPhone 14 Pro - 2022-12-20 at 09 41 01

**Comment by eviltofu** [comment]
# Colors.appActionButtonGreen

## gradientColorForTicker

1. From main wallet screen, select an item with appreciation.
2. From main wallet screen, select an item with depreciation.

Class | Before | Light After | Dark After
-|-|-|-
gradientColorForTicker (Appreciation) / TokenHistoryChartViewModel | Simulator Screen Shot - iPhone 14 - 2022-12-19 at 12 05 07 | Simulator Screen Shot - iPhone 14 Pro - 2022-12-19 at 11 41 10 | Simulator Screen Shot - iPhone 14 Pro - 2022-12-19 at 11 41 16
gradientColorForTicker (Depreciation) / TokenHistoryChartViewModel | Simulator Screen Shot - iPhone 14 - 2022-12-19 at 12 02 39 | Simulator Screen Shot - iPhone 14 Pro - 2022-12-19 at 11 40 35 | Simulator Screen Shot - iPhone 14 Pro - 2022-12-19 at 11 40 41

## actionButtonBackgroundColor/GenerateTransferMagicLinkViewModel

Unknown

Class | Before | Light After | Dark After
-|-|-|-
actionButtonBackgroundColor / GenerateTransferMagicLinkViewModel|?|?|?

## actionButtonBackgroundColor/GenerateSellMagicLinkViewModel

Unknown

Class | Before | Light After | Dark After
-|-|-|-
actionButtonBackgroundColor / GenerateSellMagicLinkViewModel|?|?|?

## actionButtonBackgroundColor/StatusViewControllerViewModel

Unknown

Class | Before | Light After | Dark After
-|-|-|-
actionButtonBackgroundColor / StatusViewControllerViewModel|?|?|?

**Comment by eviltofu** [comment]
# Colors.appActionButtonShadow

## buttonShadowColor/ButtonsBarViewModel

1. From main wallet screen, look at Buy Crypto Button.

Class | Before | Light After | Dark After
-|-|-|-
buttonShadowColor/ButtonsBarViewModel | Simulator Screen Shot - iPhone 14 - 2022-12-20 at 10 54 04 | Simulator Screen Shot - iPhone 14 Pro - 2022-12-20 at 10 55 22 | Simulator Screen Shot - iPhone 14 Pro - 2022-12-20 at 10 55 26

**Comment by eviltofu** [comment]
# Colors.appBackground

## backgroundColor/ImportMagicTokenViewControllerViewModel

Unknown

Class | Before | Light After | Dark After
-|-|-|-
backgroundColor/ImportMagicTokenViewControllerViewModel | ? | ? | ?

## backgroundColor/LocalViewModel

1. Tap settings
2. Tap Advanced
3. Tap Features
4. Enable "Is language switcher enabled"
5. Tap back arrow
6. Tap Change Language

Class | Before | Light After | Dark After
-|-|-|-
backgroundColor/LocalViewModel | original | light | dark

## view.background/TokenInstanceActionViewController

Unknown

Class | Before | Light After | Dark After
-|-|-|-
view.background/TokenInstanceActionViewController | ? | ? | ?

## view.background/ElevateWalletSecurityViewController

Unknown

Class | Before | Light After | Dark After
-|-|-|-
view.background/ElevateWalletSecurityViewController | ? | ? | ?

## view.background/SeedPhraseBackupIntroductionViewController

1. Tap settings
2. Tap back up this wallet

Class | Before | Light After | Dark After
-|-|-|-
view.background/SeedPhraseBackupIntroductionViewController | original | light | dark

## view.background/ShowSeedPhraseViewController

1. Tap settings
2. Tap show seed phrase
3. Tap Got it, show my seed phrase

Class | Before | Light After | Dark After
-|-|-|-
view.background/ShowSeedPhraseViewController | origin | light | dark

## view.background/VerifySeedPhraseViewController

1. Tap settings
2. Tap back up this wallet
3. Tap back up my wallet
4. Tap Ok I wrote this down

Class | Before | Light After | Dark After
-|-|-|-
view.background/VerifySeedPhraseViewController | original | light | dark

**Comment by eviltofu** [comment]
# Colors.appGrayLabel

## choiceLabelColor/NewTokenViewModel

Unknown

Class | Before | Light After | Dark After
-|-|-|-
choiceLabelColor/NewTokenViewModel|?|?|?

## label.textColor/TextView

1. Tap blockie
2. Tap +
3. Tap import
4. Tap keystore JSON. Look at text view (Keystore JSON)

Class | Before | Light After | Dark After
-|-|-|-
label.textColor/TextView|original|light|dark

## label.textColor/TextField

1. Tap blockie
2. Tap +
3. Tap import
4. Tap keystore JSON. Look at text field (Password)

Class | Before | Light After | Dark After
-|-|-|-
label.textColor/TextField|original|light|dark

## label.textColor/AddressTextField

1. Tap blockie
2. Tap +
3. Tap import
4. Tap watch

Class | Before | Light After | Dark After
-|-|-|-
label.textColor/AddressTextField|original|light|dark

## valueChangeValueColor()/WalletSummaryViewModel

We are testing the non-appreciation/depreciation colour.

1. Create a new empty wallet
2. Look at the "- - Today" string on the main screen.

Class | Before | Light After | Dark After
-|-|-|-
valueChangeValueColor()/WalletSummaryViewModel (No appreciation/depreciation)|original|light|dark

**Comment by eviltofu** [comment]
# Colors.appGreenContrastBackground

## alternativeAmountLabel/AmountTextField

1. Switch out of testnet if you are in testnet
2. Tap ETH on main wallet screen
3. Tap Send
4. Tap 1. Look at the USD figure

Class | Before | Light After | Dark After
-|-|-|-
alternativeAmountLabel/AmountTextField|Origin|Light|Dark

## createTokenScriptFileStatusButton/VerifiableStatusViewController

Unknown

Class | Before | Light After | Dark After
-|-|-|-
createTokenScriptFileStatusButton/VerifiableStatusViewController|?|?|?

**Comment by eviltofu** [comment]
# Colors.appHighlightGreen, Colors.apprecationGreen, Colors.apprecationRed

## valueChangeValueColor/EthCurrencyHelper

1. From main wallet screen, tap blockie
2. Look at percentile change in wallets

Class | Before | Light After | Dark After
-|-|-|-
valueChangeValueColor/EthCurrencyHelper (appreciate)|original-app|light-app|dark-app

Class | Before | Light After | Dark After
-|-|-|-
valueChangeValueColor/EthCurrencyHelper (depreciate)|original-dep|light-dep|dark-dep

## ImportMagicTokenCardRowViewModel/ImportMagicTokenViewController

Unknown

Class | Before | Light After | Dark After
-|-|-|-
ImportMagicTokenCardRowViewModel/ImportMagicTokenViewController|?|?|?

**Comment by eviltofu** [comment]
# Colors.appRed

## deleteAction.backgroundColor/AccountsViewModel

1. From the main wallet screen, tap the blockie icon
2. If there is only one account, create another one and go back to the wallet screen by tapping the blockie icon again.
3. Slide from right to left on a wallet which is not the current wallet

Class | Before | Light After | Dark After
-|-|-|-
deleteAction.backgroundColor/AccountsViewModel|original|light|dark

## deleteAction.backgroundColor/PriceAlertsViewController

Uknown

Class | Before | Light After | Dark After
-|-|-|-
deleteAction.backgroundColor/PriceAlertsViewController|?|?|?

## deleteAction.backgroundColor/BookmarksViewController

1. From main wallet screen, tap browser
2. Add a bookmark by typing in a url and tapping go and then tapping the ellipse button and select add to bookmarks
3. Tap the home button
4. Tap the "my bookmarks" button
5. Slide from right to left to delete a bookmark

Class | Before | Light After | Dark After
-|-|-|-
deleteAction.backgroundColor/BookmarksViewController|original|light|dark

## deleteAction.backgroundColor/BrowserHistoryViewController

1. From main wallet screen, tap browser
2. Enter apple.com in the url bar and tap go
3. Tap the home button
4. Tap "my history" button
5. Select and entry and slide from right to left

Class | Before | Light After | Dark After
-|-|-|-
deleteAction.backgroundColor/BrowserHistoryViewController|original|light|dark

## deleteButton.tintColor/DappViewCell

Unknown

## textFieldBorderColor/textFieldTextColor for TextField

1. Change to a wallet where you can send ETH
2. Tap ETH
3. Tap send
4. Enter valid amounts and recipient name
5. Tap send
6. Tap edit
7. Tap custom
8. Drag Gas Price all the way to the right
9. Look at the TextField

Class | Before | Light After | Dark After
-|-|-|-
textFieldBorderColor/textFieldTextColor for TextField|original|light|dark

## mask.backgroundColor/WhereIsWalletFoundOverlayView

Unknown

## statusColor/ImportMagicTokenViewControllerViewModel

Unknown

## noteTitleLabel, noteLabel, noteBorderView/SetSellTokensExpiryDateViewController

Unknown

## cancelButtonTitleColor/GenerateSellMagicLinkViewModel

Unknown

## cancelButtonTitleColor/GenerateTransferMagicLinkViewModel

Unknown

## deleteAction.backgroundColor/EnabledServersViewController

1. From main wallet screen, tap settings
2. Tap select active networks
3. Tap the + on the upper right corner
4. Select one network and add it, screen will shift back to main wallet screen after adding
5. Tap settings
6. Tap select active networks
7. Scroll down until you locate the network added
8. Select and slide on the network from right to left

Class | Before | Light After | Dark After
-|-|-|-
deleteAction.backgroundColor/EnabledServersViewController|?|?|?

**Comment by hboon** [comment]
There you go:

How to navigate to the view controllers:

* `actionButtonBackgroundColor` in `GenerateTransferMagicLinkViewModel` in <`GenerateTransferMagicLinkViewController`> / `NFTCollectionCoordinator`

Refer to https://github.com/AlphaWallet/alpha-wallet-ios/pull/5956#issuecomment-1358787748

* `actionButtonBackgroundColor` in `GenerateSellMagicLinkViewModel` in <`GenerateSellMagicLinkViewController`> / `NFTCollectionCoordinator`

Refer to https://github.com/AlphaWallet/alpha-wallet-ios/pull/5956#issuecomment-1358793677

* `backgroundColor`/<`ImportMagicTokenViewControllerViewModel`>

Never mind this ^

* `view.background`/<`TokenInstanceActionViewController`>

1. Ethereum mainnet. Use 0xbbc
2. Wallet tab
3. Enter "tokenscript" in the search bar
4. Tap on WETH (DAI or SAI should work too)
5. Tap on Wrap (or Unwrap)

* `view.background`/<`ElevateWalletSecurityViewController`>

On a device, not simulator:

1. Settings tab
2. Back up this Wallet
3. Tap Back up my Wallet
4. OK, I wrote this down
5. Tap to enter seed phrase
6. Tap Verify Seed Phrase
7. Tap Lock Seed Phrase

You can do it more easily on a simulator by modifying this to always return true: `BackupCoordinator.canPromptElevateSecurity`

* `choiceLabelColor`/<`NewTokenViewModel`>

Doesn't look like these are used anymore: `choiceLabelColor` and `choiceLabelFont` 

* `createTokenScriptFileStatusButton`/<`VerifiableStatusViewController`>

1. Enable feature flag `.isTokenScriptSignatureStatusEnabled`
2. Ethereum mainnet. Use 0xbbc
3. Wallet tab
4. Enter "tokenscript" in the search bar
5. Tap on WETH (USDC, DAI or SAI should work too). Appears at the top right hand corner

* `ImportMagicTokenCardRowViewModel`/<`ImportMagicTokenViewController`>

Never mind this ^

* `deleteAction.backgroundColor`/<`PriceAlertsViewController`>

1. Enable feature flag `.isAlertsEnabled`
2. Ethereum mainnet. Use 0xbbc
3. Wallet tab
4. Enter Ethereum mainnet
5. Enter Alerts sub-tab

* `deleteButton.tintColor`/<`DappViewCell`>

1. Browser tab
2. Visit a URL
3. Add to My Bookmarks
4. Tap home button (still in Browser tab)
5. Tap and hold the biggish favicon that should now appear below "My Bookmarks" and "My History"

* `mask.backgroundColor`/<`WhereIsWalletFoundOverlayView`>

Add this to force it to appear:

```
DispatchQueue.main.asyncAfter(deadline: .now() + 2) {
    WhereIsWalletAddressFoundOverlayView.show()
}
```

We removed it temporarily recently.

* `statusColor`/<`ImportMagicTokenViewControllerViewModel`>

Never mind this ^

* `noteTitleLabel`, `noteLabel`, `noteBorderView`/<`SetSellTokensExpiryDateViewController`>
* `cancelButtonTitleColor`/<`GenerateSellMagicLinkViewModel`>

Refer to https://github.com/AlphaWallet/alpha-wallet-ios/pull/5956#issuecomment-1358793677

* `cancelButtonTitleColor`/<`GenerateTransferMagicLinkViewModel`>

Refer to https://github.com/AlphaWallet/alpha-wallet-ios/pull/5956#issuecomment-1358787748

* `hideAction.backgroundColor`/<`AssetDefinitionsOverridesViewController`>

Never mind this ^

* `tintColor`/<`createTokenScriptFileStatusButton`>

Refer to the one for `createTokenScriptFileStatusButton`/<`VerifiableStatusViewController`>

* `amountTextColor`/<`TransactionRowViewModel`>

This one's harder to be reliable unless you know what you are looking for in the Activities tab, but an easier way is to force the old Transactions tab to appear instead by disabling the feature flag `.isActivityEnabled`. Need at least 1 transaction in the tab

* `noteTitleLabel, noteLabel, noteBorderView` / <`SetTransferTokensCardExpiryDateViewController`> [link](https://github.com/AlphaWallet/alpha-wallet-ios/pull/5956/files#diff-088de9f81bd25e3b59a5fab903971b79262cea0fa4c6e762bbe564d7e0a4af16)

Should be https://github.com/AlphaWallet/alpha-wallet-ios/pull/5956#issuecomment-1358787748 or https://github.com/AlphaWallet/alpha-wallet-ios/pull/5956#issuecomment-1358793677

* `foregroundColor,attributedDescription`/<`ElevateWalletSecurityViewModel`> [Link](https://github.com/AlphaWallet/alpha-wallet-ios/pull/5956/files#diff-f87f3b914fbe54eef36ad8c449b18ed2d342600e4744075a16286d3eb9753cd0)

Should be similar to `view.background`/<`ElevateWalletSecurityViewController`>

**Pull request closed without merging**